scripts/opdracht4.js

```diff
@@ -0,0 +1,90 @@
+// #### Opdracht 4 -Functies
+//
+// Maak deze gehele opdracht eerst alsof je het voor één tv doet. We gaan één tv weergeven in het volgende format:
+//
+//     ```
+//   Philips 43PUS6504/12 - 4K TV
+//   €379,-
+//   43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm)
+//   ```
+//
+//     * **Opdracht 4a:** Maak een herbruikbare functie die een string genereert voor de naam van één tv en deze teruggeeft in het format `[merk] [type] - [naam]` zoals `Philips 43PUS6504/12 - 4K TV` of `NIKKEI NH3216SMART - HD smart TV`.
+
+// Inventory keys aanspreken: brand, type en name, iteren door array met objects met forloop-methode want maar 1 resultaat terug. Poging twee: geen forloop maken maar gewoon simpel houden.
+
+function brandTypeName(tvObjectsArray) {
+    const oneTV = tvObjectsArray.brand + " " + tvObjectsArray.type + " - " + tvObjectsArray.name;
```

Comment:
Dit vind ik altijd wat leesbaarder
```suggestion
    const oneTV = `${tvObjectsArray.brand} ${tvObjectsArray.type} - ${tvObjectsArray.name}`;
```
Wellicht jij ook? 😉